mod/members/languages/en.php

```diff
@@ -7,5 +7,6 @@
 	'members:title:newest' => 'Newest members',
 	'members:title:popular' => 'Most popular members',
 	'members:title:online' => 'Online members',
+	'members:title:alpha' => 'Alphabetical members',
```

Comment:
I'd just call this "All members" like the All groups listing that's also sorted. Otherwise LGTM